### PR TITLE
Welcome

### PR DESCRIPTION
**Pull request by muydanny**
Added the welcome page with initial phase 1 styling with spec. Used Bootstrap.

### CODE REVIEW

File: app/views/welcome/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<h1>  Welcome to Podsmack </h1>
```
Comment (muydanny) [inline app/views/welcome/index.html.erb]: From original welcome page. Could get rid of this.

File: spec/features/welcome_page_spec.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe "Podsmack Home Page" do
```
Comment (muydanny) [inline spec/features/welcome_page_spec.rb:3]: Spec for welcome page. Will continuously add as it grows

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -10,6 +10,12 @@
   </head>
 
   <body>
+    <nav class = "topnav">
```
Comment (muydanny) [inline app/views/layouts/application.html.erb]: Deleted later

File: app/views/welcome/index.html.erb [hunk 4]
```diff
@@ -2,4 +2,27 @@
 
 <div 100px>
   <img src ="https://media.giphy.com/media/xT9IgFLfWUZigjoem4/giphy.gif">
-</div>
\ No newline at end of file
+</div>
```
Comment (muydanny) [inline app/views/welcome/index.html.erb:5]: Same with all this. Prior to bootstrap to get a feel for what we wanted.

File: Gemfile [hunk 5]
```diff
@@ -15,6 +15,7 @@ gem 'sass-rails', '>= 6'
 gem 'webpacker', '~> 4.0'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 2.7'
+gem 'bootstrap', '~> 4.5.0'
```
Comment (muydanny) [inline Gemfile:18]: Since were using rails 6 had to have the latest bootstrap

Comment (janegreene) [reply Gemfile:18]: Getting fancy!

**Pull request merged**